Jenkinsfile-test

```diff
@@ -0,0 +1,9 @@
+@Library("tdr-jenkinslib@use-sbt-with-postgres-task") _
```

Comment:
This library reference is to a specific branch. Should this be updated to point to master?

Reply:
Yep, I've changed it to point to master